addons/project_account/models/project.py

```diff
@@ -0,0 +1,136 @@
+        if res_id:
+            action['view_mode'] = 'form'
+            action['res_id'] = res_id
+        return action
+
+    def _get_action_for_profitablity_section(self, record_ids, name):
+        self.ensure_one()
+        args = [[('id', 'in', record_ids)]]
+        if len(record_ids) == 1:
+            args.append(record_ids[0])
+        return {'name': name, 'type': 'object', 'args': json.dumps(args)}
+
+    def _get_domain_aal_with_no_move_line(self):
+        """ this method is used in order to overwrite the domain in sale_timesheet module. Since the field 'project_id' is added to the "analytic line" model
+        in the hr_timesheet module, we can't add the condition ('project_id', '=', False) here. """
+        return [('account_id', '=', self.analytic_account_id.id), ('move_line_id', '=', False)]
+
+    def _get_items_from_aal(self, with_action=True):
+        domain = self._get_domain_aal_with_no_move_line()
+        aal_other_search = self.env['account.analytic.line'].sudo().search_read(domain, ['id', 'amount'])
+        if not aal_other_search:
+            return {
+                'revenues': {'data': [], 'total': {'invoiced': 0.0, 'to_invoice': 0.0}},
+                'costs': {'data': [], 'total': {'billed': 0.0, 'to_bill': 0.0}},
+            }
+        total_revenues = total_costs = 0.0
+        cost_ids = []
+        revenue_ids = []
+        for aal in aal_other_search:
+            aal_amount = aal['amount']
+            if aal_amount < 0.0:
+                total_costs += aal_amount
+                cost_ids.append(aal['id'])
+            else:
+                total_revenues += aal_amount
+                revenue_ids.append(aal['id'])
+        costs_action = None
+        revenues_action = None
+
+        if with_action and self.user_has_groups('account.group_account_readonly'):
+            costs_action = self._get_action_for_profitablity_section(cost_ids, 'action_aal')
+            revenues_action = self._get_action_for_profitablity_section(revenue_ids, 'action_aal')
+
+        # we dont know what part of the numbers has already been billed or not, so we have no choice but to put everything under the billed/invoiced columns.
+        # The to bill/to invoice ones will simply remain 0
+        profitability_sequence_per_invoice_type = self._get_profitability_sequence_per_invoice_type()
+        revenues = [{'id': 'other_revenues', 'sequence': profitability_sequence_per_invoice_type['other_revenues'],
+                         'invoiced': total_revenues, 'to_invoice': 0.0, 'action': revenues_action}]
+        costs = [{'id': 'other_costs', 'sequence': profitability_sequence_per_invoice_type['other_costs'],
+                      'billed': total_costs, 'to_bill': 0.0, 'action': costs_action}]
+
+        return {
+            'revenues': {'data': revenues, 'total': {'invoiced': total_revenues, 'to_invoice': 0.0}},
+            'costs': {'data': costs, 'total': {'billed': total_costs, 'to_bill': 0.0}},
+        }
```

Comment:
```suggestion
        revenues = {
            'id': 'other_revenues', 
            'sequence': profitability_sequence_per_invoice_type['other_revenues'],
            'invoiced': total_revenues, 
            'to_invoice': 0.0, 
            'action': revenues_action,
        }
        costs = {
            'id': 'other_costs', 
            'sequence': profitability_sequence_per_invoice_type['other_costs'],
            'billed': total_costs, 
            'to_bill': 0.0, 
            'action': costs_action,
        }

        return {
            'revenues': {'data': [revenues], 'total': {'invoiced': total_revenues, 'to_invoice': 0.0}},
            'costs': {'data': [costs], 'total': {'billed': total_costs, 'to_bill': 0.0}},
        }
```
If you want, you can format like this. 🙂